src/main/java/art/heredium/service/MembershipService.java

```diff
@@ -8,18 +9,86 @@
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
+import art.heredium.core.config.error.entity.ApiException;
+import art.heredium.core.config.error.entity.ErrorCode;
+import art.heredium.domain.coupon.entity.Coupon;
+import art.heredium.domain.coupon.model.dto.request.MembershipCouponCreateRequest;
+import art.heredium.domain.coupon.repository.CouponRepository;
 import art.heredium.domain.membership.entity.Membership;
+import art.heredium.domain.membership.model.dto.request.MembershipCreateRequest;
 import art.heredium.domain.membership.repository.MembershipRepository;
+import art.heredium.domain.post.entity.Post;
+import art.heredium.domain.post.repository.PostRepository;
 
 @Slf4j
 @Service
 @RequiredArgsConstructor
 @Transactional(rollbackFor = Exception.class)
 public class MembershipService {
 
+  private static final boolean DEFAULT_ENABLED_STATUS = true;
+  private static final Long DEFAULT_MEMBERSHIP_PERIOD = 12L;
```

Comment:
```suggestion
  private static final Long DEFAULT_MEMBERSHIP_PERIOD = 12L; // months
```
Add comment để dễ maintain về sau.

Reply:
I updated in https://github.com/Incuvers-IT/heredium-BE/pull/15/commits/0fba30355557b33808e09a5f2bfe7ed462e915cf